### DISCUSSION TITLE
Branch is out of date message is scarier than in browser

### DISCUSSION THREAD

**Discussion by jsoref** [community/community#43972] [Mobile] [answered]
https://github.com/github/docs/pull/23121
![Screenshot_20230111-100501.png](https://user-images.githubusercontent.com/2119212/211840824-90c9f773-3ca3-4190-be28-4c8de334a5e8.png)
![Screenshot_20230111-100303.png](https://user-images.githubusercontent.com/2119212/211840845-0db1ebd7-9f41-4f06-affa-7601d765e62a.png)

> ❌ Unable to merge 

> ⚠️ This branch is out-of-date with the base branch

One is a red error and the other is a gray warning.

**Comment by jacortinas** [comment, marked as answer]
Hi, @jsoref! I'm going to open an issue with the team to see how we can make this more consistent. Thank you!
